### DISCUSSION TITLE
IDENTTIFY e AUTO INCREMENT

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#141] [Tira Dúvidas] [answered]
- **Bom dia, caríssimos!**
- O IDENTIFY(1, 1) para incrementar algum item no SQL Server seria equivalente ao AUTO INCREMENT do MySQL? Abraços!

**Comment by andrebaltieri** [comment, marked as answer]
@pedro-mileipp bom dia meu caro!
Exatamente 💜
